src/Common/Wpf/Impl/Extensions/VisualTreeExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Windows;
+using System.Windows.Media;
+
+namespace Microsoft.Common.Wpf.Extensions {
+    public static class VisualTreeExtensions {
+        public static T FindChild<T>(DependencyObject o) where T : DependencyObject {
```

Comment:
I think this method requires better name.
It searches for the first child of some type in depth, correct?